installer/tue-install-impl.bash

```diff
@@ -588,7 +583,7 @@ Command: tue-install-cp $*"
             tue-install-debug "File $file and $cp_target are different, copying..."
             if "$root_required"
             then
-                tue-install-debug "Using elevated privileges (sudo)"
+                echo "Using elevated privileges (sudo) to copy ${file} to ${cp_target}"
```

Comment:
Why not use `tue-install-info`?

Reply:
Glad you asked!

Because that is really intrusive. And you get a whole summary at the end as if something really cool just happened.

But the debug is too little, because `sudo` rights are asked and I have no clue why. That's why an echo just before _every_ sudo command is good practice I guess.

Reply:
So we actually need a `tue-install-print` or something similar? As these echo's don't show in the logfile.